week_01/Meeting_Room_2.cpp

```diff
@@ -0,0 +1,28 @@
+// TC: O(NlogN)
+// MC: O(N)
+
+class Solution {
+public:
+    int minMeetingRooms(vector<Interval>& intervals) {
+        sort(intervals.begin(), intervals.end(), [](auto& a, auto& b) {
+            return a.end < b.end;
+        });
+
+        priority_queue<int, vector<int>, greater<int>> pq;
```

Comment:
If you have already sorted the array, do you still need a priority queue? What's this for?